chore: update ct how-to (#13981)
microsoft/playwright: docs/src/test-components-js.md

### Step 1: Install Playwright Test for components for your respective framework

```sh
npm init playwright@latest --ct
```

This step creates several files in your workspace:

#### `playwright/index.html`

> This file defines html file that will be used to render components during testing.

Reviewer: ```suggestion
This file defines an html file that will be used to render components during testing.
```

> It must contain element with `id="root"`, that's where components are mounted. It must

Reviewer: ```suggestion
It must contain an element with `id="root"` which is where components are mounted. It must
```

> also link the script called `playwright/index.ts`.

Reviewer: If they are using vanilla js, this seems the `.ts` seems a bit out of place!

</html>
```

#### `playwright/index.ts`

> You can include stylesheets, apply theme and inject code into the page where

Reviewer: ```suggestion
You can include stylesheets, apply theming and inject code into the page where
```

> component is mounted using this script. It can be either `.js` or `.ts` file.

Reviewer: ```suggestion
component is mounted using this script. It can be either a `.js` or `.ts` file.
```

```js
// Apply theme here, add anything your component needs at runtime here.
```

#### `playwright/playwright.d.ts`

> This file makes sure `mount` fixture is typed in your tests.

Reviewer: ```suggestion
This file makes sure the `mount` fixture is typed in your tests. `@playwright/test` ships with [default fixtures](https://playwright.dev/docs/test-fixtures#built-in-fixtures) like `browser` and `page`; `mount` is a component-testing specific fixture that's added by the Component Testing plugin.
```

Author: This might change.
